pkg/ddl/ddl.go

```diff
@@ -1205,13 +1233,18 @@ func (d *ddl) DoDDLJob(ctx sessionctx.Context, job *model.Job) error {
 		recordLastDDLInfo(ctx, historyJob)
 	}()
 	i := 0
+	notifyCh, ok := d.getJobDoneCh(job.ID)
+	if !ok {
+		// shouldn't happen, just give it a dummy one
```

Comment:
it's OK to select from `nil` channel https://golangbyexample.com/select-with-nil-channel-golang/